### PR TITLE
Import sr10 type sheets

### PR DESCRIPTION
**Pull request by ascii-dev**
## Description
Imports sheets with the same format as sr10 from the 2015 report file.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: reports/report_details.py [hunk 1]
```diff
@@ -667,6 +695,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_53': {
+        '2015': {
+            'name': '53',
+            'title': 'Internet - Female reporters in main stories',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'title': 'Internet - Sex of reporters in main stories',
```

File: reports/report_details.py [hunk 2]
```diff
@@ -951,6 +993,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_84': {
+        '2015': {
+            'name': '84',
+            'title': 'Internet - Occupation of Female news subjects',
+            'desc': 'Internet - Occupation of Female news subjects',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'desc': '',
```

**Review by kilemensi — COMMENTED** [review]
👍🏾 @ascii-dev 

In the queries sheet, queries from 79:

  1. Have 1 or 0 dropdown in the **Query Name 2020** column. Any idea what it means?, and
  2. They shouldn't have description

**Comment by ascii-dev** [comment]
> 👍🏾 @ascii-dev
> 
> In the queries sheet, queries from 79:
> 
> 1. Have 1 or 0 dropdown in the **Query Name 2020** column. Any idea what it means?, and
> 2. They shouldn't have description

1 - Yes, include this query;  0 - No, omit this query @kilemensi 

Kindly help approve this so I can merge and  include its contents in the rebase I've been working on. Thanks 🙏🏽

File: reports/report_details.py [hunk 3]
```diff
@@ -959,6 +1008,13 @@ def add_transnational_to_regions(regions):
         },
     },
     'ws_85': {
+        '2015': {
+            'name': '85',
+            'title': 'Internet - Functions of news subjects',
+            'desc': 'Internet - Functions of news subjects',
```
Comment (kilemensi) [inline reports/report_details.py]:
```suggestion
            'desc': '',
```

File: reports/historical/_base_importer.py [hunk 4]
```diff
@@ -71,6 +71,8 @@ def slurp_secondary_col_table(
             major_col_heading = canon(
                 ws.cell(column=icol, row=major_col_heading_row).value
             )
+            if not major_col_heading:
+                continue
```
Comment (kilemensi) [inline reports/historical/_base_importer.py:75]: When would this normally happen?

**Review by kilemensi — APPROVED** [review]
:rocket: @ascii-dev 

Lots of sheets also need the N heading updated.

File: reports/historical/_base_importer.py (hunk 4 above)
Comment (ascii-dev) [reply reports/historical/_base_importer.py:75]: For some of the sheets, there are spaces in between the columns which stretch the entire height of the data, for those, we need to skip them entirely, which was why I added that line @kilemensi

**Pull request merged**